packages/trackerless-network/test/integration/NodeInfoRpc.test.ts

```diff
@@ -89,12 +89,16 @@ describe('NetworkStack NodeInfoRpc', () => {
                 {
                     id: streamPartId1,
                     controlLayerNeighbors: [normalizePeerDescriptor(otherPeerDescriptor)],
-                    contentDeliveryLayerNeighbors: [normalizePeerDescriptor(otherPeerDescriptor)]
+                    contentDeliveryLayerNeighbors: [{
+                        peerDescriptor: normalizePeerDescriptor(otherPeerDescriptor)
+                    }]
```

Comment:
Could we assert that there is a `rtt` value? (also at line `100`)

Reply:
That would require waiting for neighbor updates and assuming that the RTT is always some value in the integration test. This could cause flakiness.